### PR TITLE
OIPIP-0004

### PR DESCRIPTION
**Pull request by daviortega**
Change the name of the OIP Object from `artifact` to `record`

### CODE REVIEW

File: oipip-0004.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+Artifact can be misinterpreted. The second definition in the Merriam-Webster dictionary is:
+
+```
+2.a : a product of artificial character (as in a scientific test) due usually to extraneous (such as human) agency
+2.c : a defect in an image (such as a digital photograph) that appears as a result of the technology and methods used to create and process the image 
+```
+
+This is particularly bad for scientific use of OIP.
+
```
Comment (cchrysostom) [inline oipip-0004.md]: Also explain why *record* is preferred. Something like common definitions include the idea of something that registers something like evidence into a permanent medium.

Comment (daviortega) [reply oipip-0004.md]: Done

**Review by cchrysostom — APPROVED** [review]

**Comment by copernicus-mogley** [comment]
A little background on why "artifact" was chosen: The term comes from communications/media branch of academia and social sciences in which a cultural/social artifact is defined as anything created by humans which gives information about the culture of its creator and users. This includes songs, television episodes, movies, books, etc. While I see that this doesn't fit scientific records, perhaps the sub-type for those artifacts could be "records" instead of changing the broad term to something which does not fit other types of media published to the index.

A couple of sources (sorry that I don't have time to dig up more):
https://ivyroseroberts.wordpress.com/2015/08/22/what-is-a-media-artifact/
https://en.wikipedia.org/wiki/Cultural_artifact

**Comment by cchrysostom** [comment]
Thanks, @copernicus-mogley , it is good to know the history. It feels like the scope of the "Index" has grown. For property rights, **record** makes sense. In this field the idea of a **record** comes from  library science, which does feel like primary concept of the *Worldwide Public Record*.

Library Science definition: http://cushing.library.tamu.edu/services/records_mgmt/what_is_a_record.html .

**Comment by daviortega** [comment]
HI @copernicus-mogley, as @cchrysostom said, thank you very much for the input. Would you mind if I include it in the main text of the Proposal?

Also, I took a bit to respond to this because I was giving a long thought about it. I liked the term `Artifact` a lot when I first saw it, but Artifact is, as you said, the song, television episodes, movies and etc, not the record of the artifact. In the current implementation of OIP, the artifact _per se_, the payload of the content, goes to IPFS and the record of it is an OIP Object - the JSON formatted Object that goes to the FLO blockchain.

I understand that the term Artifact is broad and well thought out, but it restrict the use in scientific research a lot as the IP text says, because it literally means "data mistake".

I have been talking about ETDB in at least 4 different academic context and every single time I need to stop and spend 5 minutes re-adapting people's mind that Artifact isn't what they expect to be. Literally, Artifact is the word of choice to describe a part of that data that is artificial rather than real. Many people leave the talk thinking that we are publishing a collection of tomograms with problems in it, no matter how clear I make my explanation.

Maybe `Record` isn't the best, although I do think so, but I am open for suggestions.

And as a final thought, we can keep calling the payload that goes to the IPFS as "digital artifact" and the OIP Object as "record" without loss of clarity in my opinion. That way, if a field isn't happy with the name "artifact", they can call their digital content whatever they want: tomography datasets, for example.

In contrast, the name of the OIP Object must be used by all different type of digital content and therefore, I believe it should be less conflicting as possible with the jargon of as many fields as possible.

**Comment by copernicus-mogley** [comment]
Cool. Feel free to include anything from my comment in the main text.

**Review by OstlerDev — APPROVED** [review]

**Pull request merged**